doc/source/language/variables.rst

```diff
@@ -234,8 +235,9 @@ expression, the expression will not get executed if the calling
 function had an argument present in that position.  The expression
 only gets executed if the system needed to pad a missing argument.
 
-.. versionadded:: 0.18.3
-   Optional Parameters were added as a new feature in kOS 0.18.3
+.. note::
+    .. versionadded:: 0.18.3
+        Optional Parameters were added as a new feature in kOS 0.18.3
```

Comment:
Actually, `.. versionadded::` appears to be behaving as expected.  It's only configured to italicize and add the text "New in version" before the version number.  I have long suspected that we could simply modify the configuration to use a similar style as `.. note::` (though different so that it stands out).  It has been on my list to see if that's actually possible.

I do feel like there is probably a point at which we can simply remove these blocks, for instance there have been 6 releases and 6 months since v0.18.3 was released.  Maybe the note can be considered no longer necessary.